package.json

```diff
@@ -10,19 +10,23 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "codemirror": "^5.54.0",
+    "fsh-sushi": "^0.14.0",
+    "null-loader": "^4.0.0",
     "react": "^16.13.1",
     "react-codemirror2": "^7.2.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "tar-stream": "^2.1.3",
+    "temp": "^0.9.1"
```

Comment:
I think you can do an `npm uninstall temp` since you removed the only place it was used!

Reply:
I just pushed some changes I got to package-lock.json, I'm not sure why I got them and you didn't!